tests/unit/amo/components/TestRatingsByStar.js

```diff
@@ -218,25 +246,38 @@ describe(__filename, () => {
       addon,
       location: createFakeLocation({ query: { src } }),
     });
-    const counts = root.find('.RatingsByStar-count').find(Link);
+    const tableRow = root.find('.RatingByStar-table-row');
 
-    function validateLink(link, score) {
-      expect(link).toHaveProp(
-        'to',
-        reviewListURL({ addonSlug: addon.slug, score, src }),
+    const clientApp = CLIENT_APP_ANDROID;
+    const lang = 'en-US';
+
+    function validateLink(row, score) {
+      expect(row).toHaveProp('onClick');
```

Comment:
I don't think we need this assertion. We already have one test that checks that the `onClick` prop exists when expected, and the next assertion will fail if the `onClick` prop doesn't exist.

Reply:
I will remove this in the next update.